### PR TITLE
fix: remove squash on configtls.TLSClientSetting for AWS components

### PR DESCRIPTION
**Pull request by hyunuk**
**Description:**
This PR removes squash on`configtls.TLSClientSetting` and update corresponding yaml files to ensure that the unit test passed.

**Link to tracking Issue:**
#5433

### CODE REVIEW

**Review by bogdandrutu — COMMENTED** [review]
Consider to split into a PR that fixes the broken readmes and other that change the other components to not squash

**Comment by hyunuk** [comment]
@bogdandrutu I split them into the first two commits in this PR. 
But if they are splitted into two PRs, each of them cannot pass `build-and-test/unittest(pull_request)` in Github actions, which is labeled as `Required`.

File: exporter/datadogexporter/example/example_k8s_manifest.yaml [hunk 1]
```diff
@@ -33,7 +33,8 @@ data:
     exporters:
       otlp:
         endpoint: "otel-collector.default:4317"
-        insecure: true
+        tls:
+          insecure: true
```
Comment (bogdandrutu) [inline exporter/datadogexporter/example/example_k8s_manifest.yaml:37]: This his can be split.

File: exporter/opencensusexporter/README.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ The following settings are required:
...
 [here](https://github.com/grpc/grpc/blob/master/doc/naming.md)
 
-By default, TLS is enabled and must be configured under `tls`:
+By default, TLS is enabled and must be configured under `tls:`:
 
 - `insecure` (default = `false`): whether to enable client transport security for
   the exporter's connection.
 
-As a result, the following parameters are also required:
+As a result, the following parameters are also required under `tls:`:
```
Comment (bogdandrutu) [inline exporter/opencensusexporter/README.md:21]: This his can be split.

File: exporter/zipkinexporter/README.md [hunk 3]
```diff
@@ -12,12 +12,12 @@ The following settings are required:
...
 - `format` (default = `JSON`): The format to sent events in. Can be set to `JSON` or `proto`.
 
-By default, TLS is enabled and must be configured under `tls`:
+By default, TLS is enabled and must be configured under `tls:`:
 
 - `insecure` (default = `false`): whether to enable client transport security for
   the exporter's connection.
 
-As a result, the following parameters are also required:
+As a result, the following parameters are also required under `tls:`:
```
Comment (bogdandrutu) [inline exporter/zipkinexporter/README.md:20]: This his can be split.

File: extension/oauth2clientauthextension/README.md [hunk 4]
```diff
@@ -17,10 +17,10 @@ extensions:
...
     # tls settings for the token client
     tls:
-        insecure: true
-        ca_file: /var/lib/mycert.pem
-        cert_file: certfile
-        key_file: keyfile
+      insecure: true
+      ca_file: /var/lib/mycert.pem
+      cert_file: certfile
+      key_file: keyfile
```
Comment (bogdandrutu) [inline extension/oauth2clientauthextension/README.md:23]: This his can be split.

**Review by anuraaga — APPROVED** [review]

**Review by bogdandrutu — CHANGES_REQUESTED** [review]
See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/5466#pullrequestreview-766075718

**Review by Aneurysm9 — APPROVED** [review]

**Comment by bogdandrutu** [comment]
Consider to split per component: splunk/observiq/elastic/aws, since this has approver from aws owners this may contain the aws changes, and create other 3 PRs. We need component owners to ack these changes.

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**